content/docs/user-guide/interactivity/physics/debugging.md

```diff
@@ -77,12 +78,10 @@ You must enable the [ImGui Gem](/docs/user-guide/gems/reference/debug/imgui) to
 1. Press the **Home** key to open the **ImGui** tool. The **PhysX Debug** menu appears under the **Perspective** viewport.
 
 1. Click **PhysX Debug**.
-**Example**
 
 ![PhysX Debug menu in gameplay mode.](/images/user-guide/physx/physx-debugger-imgui-tool.png)
 
-1. You can make the following changes.
-****
+You can make the following changes.
```

Comment:
It seems like the content that should be here wasn't properly transferred over from the Lumberyard docs. You can see the original page here: https://docs.aws.amazon.com/lumberyard/latest/userguide/debugging-physx.html

Since the IMGui tool in O3DE Editor still displays the settings listed in the original page, I think it's safe to copy that table over to this page. (See [Style Guide](https://www.o3de.org/docs/contributing/to-docs/style-guide/format/#tables) on formatting tables in Markdown).